pkg/server/container_stats_list_windows.go

```diff
@@ -82,15 +101,24 @@ func (c *criService) getSandboxMetrics(
 			return nil, err
 		}
 		if s, ok := v.(*runhcsstats.Statistics); ok {
-			cs.Cpu = &runtime.CpuUsage{
-				Timestamp:            stats.Timestamp.UnixNano(),
-				UsageCoreNanoSeconds: &runtime.UInt64Value{s.VM.Processor.TotalRuntimeNS},
-			}
-			cs.Memory = &runtime.MemoryUsage{
-				Timestamp:       stats.Timestamp.UnixNano(),
-				WorkingSetBytes: &runtime.UInt64Value{s.VM.Memory.WorkingSetBytes},
-			}
+			timestamp := stats.Timestamp.UnixNano()
+			cs.Cpu = getResultsCpuUsage(timestamp, s.VM.Processor.TotalRuntimeNS)
+			cs.Memory = getResultsMemoryUsage(timestamp, s.VM.Memory.WorkingSetBytes)
 		}
 	}
 	return cs, nil
 }
+
+func getResultsCpuUsage(timestamp int64, cpuUsageNanoSeconds uint64) *runtime.CpuUsage {
```

Comment:
Can we just declare these structs inline instead of a helper? The helper doesn't gain us any valuable code logic

Reply:
Done